pre-check/v1.1.x/check.sh

```diff
@@ -0,0 +1,310 @@
+#!/bin/bash -e
+
+check_failed=0
+
+record_fail()
+{
+    check_failed=$((check_failed+1))
+}
+
+
+check_bundles()
+{
+    echo ">>> Check all bundles ready..."
+
+    # fleet bundles should be ready
+    # except mcc-harvester, which has Modified status in 1.0.3
+    bundles=$(kubectl get bundles.fleet.cattle.io -A -o yaml)
+    pending=$(echo "$bundles" | yq '.items | any_c(.metadata.name != "mcc-harvester" and .spec.helm != null and .status.summary.ready == 0)')
+
+    if [ "$pending" = "false" ]; then
+      echo "All Helm bundles are ready."
+      return
+    fi
+
+    echo "There are non-ready Helm bundles:"
+    echo "$bundles" | yq '.items[] | select(.spec.helm != null and .status.summary.ready == 0) | .metadata.namespace + "/" + .metadata.name'
+    record_fail
+}
+
+check_harvester_bundle()
+{
+    echo ">>> Check the Harvester bundle is ready..."
+
+    current_summary=$(mktemp)
+    kubectl get bundles.fleet.cattle.io/mcc-harvester -n fleet-local -o yaml | yq '.status.summary' > $current_summary
+
+    expected_summary=$(mktemp)
+cat > $expected_summary <<EOF
+desiredReady: 1
+ready: 1
+EOF
+
+    if ! diff <(yq -P 'sort_keys(..)' $current_summary) <(yq -P 'sort_keys(..)' $expected_summary); then
```

Comment:
the diff is also doing a print, so L45 `cat $current_summary` may seem redundant.

Reply:
Fixed! Thanks